### ISSUE TITLE
preview7 regression: Nullable`1 types treated as required

### ISSUE THREAD

**Issue by Zenexer** [dotnet/efcore#16894]
I have a model with a `Nullable<DateTime>` that looks something like this:

``` csharp
public class Model
{
    public DateTime? DeletedAt { get; set; }
}
```

`DateTime` is a value type, so `DateTime?` is a `Nullable<DateTime>`, not a NRT.  `DeletedAt` is logically initialized to `null` and isn't required.  However, `HasData` results in the following exception:

> The seed entity for entity type 'Model' cannot be added because there was no value provided for the required property 'DeletedAt'.

**Comment by Zenexer** [comment]
Temporary workaround:

``` csharp
ModelBuilder.Entity<Model>(e =>
{
    e.Property(m => m.DeletedAt).IsRequired(false);
    // ...
});

**Comment by Zenexer** [comment]
This may have the same underlying cause as #16760, but it pertains to nullable value types instead of NRTs, so I'm not positive.  I haven't tested with nightly yet.

**Comment by ajcvickers** [comment]
@Zenexer Please post a small, runnable project/solution or complete code listing that demonstrates the behavior you are seeing.

**Comment by Zenexer** [comment]
@ajcvickers Before I devote considerable time to making a minimal project, do you happen to know whether this was likely fixed along with #16760?

**Comment by ajcvickers** [comment]
@Zenexer From the minimal information above, I would guess yes, but it's a very low confidence guess.

**Comment by ajcvickers** [comment]
**EF Team Triage:** Closing this issue as the requested additional details have not been provided and we have been unable to reproduce it.

*BTW this is a canned response and may have info or details that do not directly apply to this particular issue. While we'd like to spend the time to uniquely address every incoming issue, we get a lot traffic on the EF projects and that is not practical. To ensure we maximize the time we have to work on fixing bugs, implementing new features, etc. we use canned responses for common triage decisions.*

**Comment by Zenexer** [comment]
Confirmed fixed by #16781 in preview8.
